backend/wmg/data/schemas/corpus_schema.py

```diff
@@ -19,6 +19,9 @@
 
 uint32_domain = (np.iinfo(np.uint32).min, np.iinfo(np.uint32).max - 1)
 
+# TODO: also define and use constants for obs and var array names
```

Comment:
Do you mean turning obs and var into constants? or using a more informative constant name? do you think cells and genes are too specific as replacements?

Reply:
Those names certainly make sense! I did this refactoring on a new [branch/PR](https://github.com/chanzuckerberg/single-cell-data-portal/pull/2377), but just saw this comment now. We can rename again there, so if you want to let's take the thread over to that PR. Resolving here.